tests/testthat/test-nm-tables.R

```diff
@@ -0,0 +1,40 @@
+
+withr::local_options(list("bbr.verbose" = FALSE))
+
+test_that("nm_tables() works", {
+  res <- nm_tables(MOD1)
+  expect_equal(length(res), length(MOD1_TABLE_FILES) + 1)
+  expect_equal(res$data, nm_data(MOD1))
+  expect_equal(res$tab, nm_tab(MOD1))
+  expect_equal(res$par.tab, nm_par_tab(MOD1))
+})
+
+test_that("nm_tables() works with .files argument [BBR-NMT-001]", {
```

Comment:
You might be planning to add test IDs later, but seeing this one made me think that perhaps you forgot to add them to the other tests in this file.